libdnf/dnf-sack.cpp

```diff
@@ -1704,12 +1667,10 @@ dnf_sack_load_system_repo(DnfSack *sack, HyRepo a_hrepo, int flags, GError **err
     DnfSackPrivate *priv = GET_PRIVATE(sack);
     Pool *pool = dnf_sack_get_pool(sack);
     char *cache_fn = dnf_sack_give_cache_fn(sack, HY_SYSTEM_REPO_NAME, NULL);
```

Comment:
I think we can remove cache_fn definition too. It is created and freed without any real usage.